lauren_hw.md

```diff
@@ -0,0 +1,31 @@
+## Summer Goals:
+**Learn general skills:**    
```

Comment:
Do a markdown preview of this file - it may not render with the returns you expect. Eg

```
Git - analysis, lab notebook  
Shell/bash scripting  
Data analysis - R/Julia  
```

becomes:

Git - analysis, lab notebook Shell/bash scripting  Data analysis - R/Julia  

To make line breaks, either put a newline in between (paragraph break) or put two spaces at the end of a line. You can also make it a bulleted list, eg:

```
**Learn general skills:**

- Git - analysis, lab notebook  
- Shell/bash scripting  
- Data analysis - R/Julia
```